src/services/notifications.rs

```diff
@@ -24,13 +25,13 @@ pub async fn delete_registration(
         safe_address
     )?;
 
-    let response = client
+    client
         .delete(url)
         .timeout(Duration::from_millis(default_request_timeout()))
         .send()
         .await?;
 
-    forward_error(response).await
```

Comment:
Why was this required before?

Reply:
It was not required. That method was only returning `Ok(())` if the response was success or mapping the error response to an `ApiError` which in the case of this method, would be done by the `responder` implementation for `ApiError` anyway.

This call was totally redundant.